src/magnify/apps/core/models.py

```diff
@@ -222,17 +237,26 @@ def jitsi_name(self):
 
     def is_administrator(self, user):
         """Check if a user is administrator of the room."""
-        if not user.is_authenticated:
+        if not user or not user.is_authenticated:
             return False
 
         return (
-            self.user_accesses.filter(is_administrator=True, user=user).exists()
+            self.user_accesses.filter(
+                role__in=[UserRoleChoices.ADMIN, UserRoleChoices.OWNER], user=user
```

Comment:
```suggestion
                role__in=[UserRoleChoices.ADMIN, UserRoleChoices.OWNER], user=user
```
the question is administrator but there with this he could only be the owner, is this correct? You have the method is_owner as well